README.adoc

```diff
@@ -119,6 +119,14 @@ can act as starting points or examples:
  provider such as https://stormpath.com/[Stormpath], Active Directory,
  https://developers.google.com/identity/[Google], http://openid.net/[OpenID], etc.
 
+ https://github.com/SMARTRACTECHNOLOGY/devkit-simple-service-example[Create Your Own Microservice]::
```

Comment:
No, I mean moving the adoc file to this repository, into the `guides` directory.